filebeat/module/iis/w3svc/manifest.yml

```diff
@@ -0,0 +1,17 @@
+module_version: 1.0
+
+var:
+  - name: paths
+    default:
+      - C:/inetpub/logs/LogFiles/*/*
+    os.windows:
+      - C:/inetpub/logs/LogFiles/*/*
+
```

Comment:
I assume here you have to define something for mac and linux (empty) similar to https://github.com/elastic/beats/blob/master/filebeat/module/auditd/log/manifest.yml#L7 @tsg knows more here.

Reply:
I assumed it was unnecessary considering IIS is a windows-only application.